SYCL/AtomicRef/add-native.cpp

```diff
@@ -1,11 +1,11 @@
 // RUN: %clangxx -fsycl -fsycl-unnamed-lambda -DSYCL_USE_NATIVE_FP_ATOMICS \
 // RUN: -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: %HOST_RUN_PLACEHOLDER %t.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out -Xsycl-target-backend=nvptx64-nvidia-cuda-sycldevice --cuda-gpu-arch=sm_60
```

Comment:
I believe the arg is being added to the wrong line - the Clang compilation command is what should receive it. The same applies to the `fetch_sub` test.
Meanwhile, the fact that this CUDA-specific arg is passed regardless of the actual GPU target isn't of much bother - it's good that we can rely on the driver to simply ignore it for generic compilations.

Reply:
I would also suggest splitting the resulting `RUN` command into more lines so that the format adhered to.

Reply:
Thanks for the prompt review, @AGindinson.  Both fixed in 0a01ee8.